ramp-frontend/ramp_frontend/tests/test_ramp.py

```diff
@@ -411,6 +413,119 @@ def test_ask_for_event_mail(client_session):
                         in outbox[0].body)
 
 
+@pytest.mark.parametrize(
+    "submission_dir, file_name, correct",
```

Comment:
```suggestion
    "submission_dir, filename, correct",
```

Reply:
I would write 2 separate tests. One for when we expect an error and one when we expect to work.